tests/fixtures/ppr/app/dynamic-params/[id]/page.js

```diff
@@ -0,0 +1,44 @@
+import { Suspense } from 'react'
+import { connection } from 'next/server'
+
+// no `generateStaticParams` export
+
+async function getData(id) {
+  await connection()
+  const res = await fetch(`https://api.tvmaze.com/shows/${id}`, {
+    next: {
+      tags: [`show-${id}`],
+    },
+  })
+  await new Promise((res) => setTimeout(res, 3000))
+  return res.json()
+}
+
+async function Content({ id }) {
+  const data = await getData(await id)
+
+  return (
+    <>
+      <h1>Dynamic Page (dynamic params): {id}</h1>
```

Comment:
As [documented here](https://nextjs.org/docs/app/api-reference/file-conventions/dynamic-routes#with-cache-components), `await params` must be wrapped in a Suspense boundary when using Cache Components and no `generateStaticParams`, otherwise the build fails with https://nextjs.org/docs/messages/blocking-route.

AFAICT, this is not strictly related to the bug this PR is fixing. It's just another incidental difference between dynamic segments with/without `generateStaticParams`.